### PR TITLE
Hide placements from the assessment only route, fix some bugs by creating a new macro and add some content about the minimum number of places required.

### PR DESCRIPTION
**Pull request by jesseyuen**

### CODE REVIEW

File: app/assets/sass/components/_notice-banner.scss [hunk 1]
```diff
@@ -1,19 +1,60 @@
...
+
+// .notice-banner .govuk-error-summary__list a:hover {
+//   color: $govuk-brand-colour;
+// }
+
+// .notice-banner .govuk-error-summary__list,
+// .notice-banner .govuk-error-summary__list a:link, 
+// .notice-banner .govuk-error-summary__list a:visited {
+//   color: $govuk-brand-colour;
+// }
```
Comment (edwardhorsford) [inline app/assets/sass/components/_notice-banner.scss]: Can these comments be removed?

File: app/lib/utils.js [hunk 2]
```diff
@@ -158,24 +158,35 @@ exports.recordIsComplete = record => {
   return recordIsComplete
 }
 
+// Check if the placment criteria has been met
```
Comment (edwardhorsford) [inline app/lib/utils.js]: Comment needs updating?

File: app/views/_components/notice-banner/template.njk [hunk 3]
```diff
@@ -0,0 +1,16 @@
+<div class="app-notice-banner 
+  {%- if params.classes %} {{ params.classes }}{% endif %}"
+  aria-labelledby="app-notice-banner-title" role="alert">
```
Comment (edwardhorsford) [inline app/views/_components/notice-banner/template.njk]: Given our usage, do you think `role="alert"` is correct?

File: app/views/_includes/actions-banner.njk [hunk 4]
```diff
@@ -1,20 +1,18 @@
 {% set placementCount = record.placement.items | length %}
 {% set hasCommencementDate = record.trainingDetails.commencementDate | length > 0 %}
 {% set minPlacementsRequired = data.settings.minPlacementsRequired %}
```
Comment (edwardhorsford) [inline app/views/_includes/actions-banner.njk:3]: Is this line needed any more?

File: app/views/_includes/actions-banner.njk [hunk 5]
```diff
@@ -1,20 +1,18 @@
 {% set placementCount = record.placement.items | length %}
```
Comment (edwardhorsford) [inline app/views/_includes/actions-banner.njk:1]: Is this line needed any more?

**Review by edwardhorsford — CHANGES_REQUESTED** [review]

File: app/assets/sass/components/_notice-banner.scss (hunk 1 above)
Comment (jesseyuen) [reply app/assets/sass/components/_notice-banner.scss]: Doh, yes.

File: app/lib/utils.js (hunk 2 above)
Comment (jesseyuen) [reply app/lib/utils.js]: Besides the spelling and grammer I think the general vibe of this one is ok? 🤔

File: app/views/_components/notice-banner/template.njk (hunk 3 above)
Comment (jesseyuen) [reply app/views/_components/notice-banner/template.njk]: Good call.

File: app/views/_includes/actions-banner.njk (hunk 5 above)
Comment (jesseyuen) [reply app/views/_includes/actions-banner.njk:1]: Yes as we're still referencing `placementCount` within `appNoticeBanner` in this file.

File: app/views/_includes/actions-banner.njk (hunk 4 above)
Comment (jesseyuen) [reply app/views/_includes/actions-banner.njk:3]: Yes as we're still referencing `minPlacementsRequired` within the `appNoticeBanner` in this file.

Comment (edwardhorsford) [reply app/views/_includes/actions-banner.njk:3]: So we are. Ignore me then!

File: app/lib/utils.js [hunk 6]
```diff
@@ -158,24 +158,35 @@ exports.recordIsComplete = record => {
...
   data = Object.assign({}, (data || this.ctx.data || false))
   
   let hasOutstandingActions = false
   let traineeStarted = record?.trainingDetails?.commencementDate
-  let placementCount = (record?.placement?.items) ? record.placement.items.length : 0
-  let minPlacementsRequired = data.settings.minPlacementsRequired
-  let needsPlacementDetails = (record?.placement?.status != 'Complete') || (placementCount < minPlacementsRequired)
-  
-  // TODO Use this to test if placements are required
-  // exports.requiresSection(record, 'placement')
```
Comment (edwardhorsford) [inline app/lib/utils.js:173]: These can be removed.

**Review by edwardhorsford — APPROVED** [review]

**Pull request merged**